resources/js/components/partials/form/Input.vue

```diff
@@ -67,6 +68,10 @@ export default {
       type: String,
       default: ''
     },
+    maxlength: {
+      type: Number,
+      default: -1,
```

Comment:
What does -1 do?

Reply:
`-1` means no max.
Maybe it's better to set it to `null` according to https://developer.mozilla.org/en/docs/Web/HTML/Element/input/text#maxlength or https://www.w3schools.com/tags/att_input_maxlength.asp